### ISSUE TITLE
Question: how to access a graph node knowing the label?

### ISSUE THREAD

**Issue by telatin** [ebran/grim#32]
I don't understand how to access a graph having the label in a string.
```
import  grim
import sequtils


var g = newGraph("my graph")

let node = g.addNode("F4", %(Id: "none", desc: "OK"))
let dest = g.addNode("F8", %(Id: "dest"))

# get node knowing label?
let label1 = "F4"
echo g.node(label1)  # <--- this does not work:
```
This script returns:
```
unhandled exception: key not found: F4 [KeyError]
```

Probably the label is not the correct attribute as it's not necessarily unique (it is in my example), but then I'd appreciate an advice on how to tag a node with a unique identifier to be used later.

Thanks!
Andrea

Note: originally posted in [StackOverflow](https://stackoverflow.com/questions/63484904/how-to-retrieve-a-node-by-label-in-a-grim-graph)

**Comment by telatin** [comment]
Answered in https://stackoverflow.com/questions/63484904/how-to-retrieve-a-node-by-label-in-a-grim-grap

TLDR:
```
let node = g.addNode("label", %(Id: "none", desc: "OK"), oid="uniqueID")
```
